src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPanelMapping.java

```diff
@@ -100,6 +112,10 @@ public static List<String> getPanelSpecialisms(@Valid SscsCaseData caseData, Ses
         if (isNull(sessionCategoryMap)) {
             return panelSpecialisms;
         }
+        // if benefit is child support specialism should be empty
+        if (isNotBlank(caseData.getBenefitCode()) && caseData.getBenefitCode().equals(CHILD_SUPPORT.getBenefitCode())) {
```

Comment:
What about IIDB ?

Reply:
In the task for IIDB it is asked to put all the selected panel specialisms. That is why I did not control IIDB. ( https://tools.hmcts.net/jira/browse/SSCS-10784 )